t/lib/apisix/plugins/jwt-auth.lua

```diff
@@ -0,0 +1,122 @@
+--
```

Comment:
please use https://jwt.io/#debugger-io to make your test data

Reply:
I modified the PR to use https://jwt.io/#debugger-io to make test data in most cases.